### PR TITLE
Tempus: update `gamma` for SSP2_222

### PR DESCRIPTION
**Pull request by sconde**
@trilinos/tempus 

## Motivation

use non-default `gamma` for `SSP2_222` to get a 2nd order, A-Stable scheme with lower principal error norm that is consistent with the app.

## Testing

Tempus/IMEX RK test has been updated. App tests using the alias has all tests passing.

### CODE REVIEW

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp [hunk 1]
```diff
@@ -142,7 +142,65 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
     this->setStepperType("IMEX RK 1st order");
     this->setOrder(1);
 
-  } else if (stepperType == "IMEX RK SSP2" || stepperType == "SSP2_222" ) {
+  } else if ( stepperType == "SSP1_111" )
```
Comment (krcb) [inline packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: @sconde Could you add some more comments from a time integrator perspective as to the differences between this method and the IMEX RK 1st order method above?

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp [hunk 2]
```diff
@@ -154,6 +212,19 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
 
     this->setStepperType("IMEX RK SSP2");
     this->setOrder(2);
+  } else if (stepperType == "SSP2_222" ) {
```
Comment (krcb) [inline packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: @sconde Likewise here

File: packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp [hunk 3]
```diff
@@ -153,16 +153,16 @@ TEUCHOS_UNIT_TEST(IMEX_RK, VanDerPol)
   stepperOrders.push_back(1.07964);
   stepperOrders.push_back(1.07964);
   stepperOrders.push_back(2.00408);
-  stepperOrders.push_back(2.00408); //SSP2_222
+  stepperOrders.push_back(2.76941); //SSP2_222
```
Comment (krcb) [inline packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp]: @sconde Are tests needed for the SSP1_111 scheme?

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp [hunk 4]
```diff
@@ -142,7 +142,64 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
...
+      // Fill c:
+      c(0) = one;
+
+      auto impTableau = Teuchos::rcp(new RKButcherTableau<Scalar>(
+        "Implicit Tableau - SSP1_111",
+        A,b,c,order,order,order));
+
+      this->setImplicitTableau(impTableau);
+    }
+    this->setStepperType("IMEX RK 1st order (SSP1_111)");
```
Comment (sconde) [inline packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: @krcb This description shows that this scheme is a single stage method and not 2 stage

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp [hunk 5]
```diff
@@ -154,6 +211,19 @@ void StepperIMEX_RK<Scalar>::setTableaus(std::string stepperType,
...
+    auto stepperERK = Teuchos::rcp(new StepperERK_Trapezoidal<Scalar>());
+    this->setExplicitTableau(stepperERK->getTableau());
+
+    // Implicit Tableau
+    auto stepperSDIRK = Teuchos::rcp(new StepperSDIRK_2Stage3rdOrder<Scalar>());
+    stepperSDIRK->setGammaType("gamma");
+    stepperSDIRK->setGamma( 0.5 );
+    this->setImplicitTableau(stepperSDIRK->getTableau());
+
+    this->setStepperType("IMEX RK SSP2( A-Stable, gamma = 0.5) ");
```
Comment (sconde) [inline packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: @krcb is this good?

File: packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp [hunk 6]
```diff
@@ -142,27 +142,27 @@ TEUCHOS_UNIT_TEST(IMEX_RK, VanDerPol)
...
   stepperOrders.push_back(2.70655);
   stepperOrders.push_back(2.00211);
   stepperOrders.push_back(2.00211);
 
   std::vector<double> stepperErrors;
   stepperErrors.push_back(0.0046423);
-  stepperErrors.push_back(0.0046423);
+  stepperErrors.push_back(0.103569); // SSP1_111
```
Comment (sconde) [inline packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp]: I had already added the test

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp (hunk 4 above)
Comment (krcb) [reply packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: Looks good

File: packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp (hunk 5 above)
Comment (krcb) [reply packages/tempus/src/Tempus_StepperIMEX_RK_impl.hpp]: Yep

File: packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp (hunk 6 above)
Comment (krcb) [reply packages/tempus/test/IMEX_RK/Tempus_IMEX_RKTest.cpp]: Got it.

**Pull request merged**